Source/JavaScriptCore/dfg/DFGPlan.cpp

```diff
@@ -428,6 +429,8 @@ Plan::CompilationPath Plan::compileInThreadImpl()
         RUN_PHASE(performCFA);
         RUN_PHASE(performGlobalStoreBarrierInsertion);
         RUN_PHASE(performStoreBarrierClustering);
+        if (Options::useMovHintRemoval())
+            RUN_PHASE(performMovHintRemoval);
```

Comment:
Why did we add the new phase here? Is it because it's need to after CFA phase, but why after `performGlobalStoreBarrierInsertion` and `performStoreBarrierClustering`?

Reply:
Just restoring the previously existing phase. We should do this at very end but this place is OK.